src/ansys/units/unit.py

```diff
@@ -77,8 +77,6 @@ def __init__(
             self._dimensions = Dimensions(_dimensions)
             if dimensions and self._dimensions != dimensions:
                 raise InconsistentDimensions()
-            if not self._dimensions:
```

Comment:
looking at this code again, it seems that we should be able to process the input unit string and remove any `<unit>^0`?

Reply:
I'm trying to achieve this using existing functions because we don't want to use string manipulation to remove any `<unit>^0`.

Reply:
@seanpearsonuk  Done. We have used `_condense` free function to remove `<unit>^0`.

Reply:
🥇